projects/plugins/jetpack/extensions/blocks/sharing-button/components/social-icons.jsx

```diff
@@ -0,0 +1,410 @@
+/* eslint-disable react/forbid-elements */
+export function getSocialIcon( service ) {
+	let svg_val = '';
+
+	switch ( service ) {
+		case 'facebook':
+			svg_val = (
+				<svg
```

Comment:
Yeah, I had troubles in images showing up in Simple sites and thought it's because using that lib. But it seem actually not, so I will revert back to social-logos